README.md

```
@@ -27,59 +28,34 @@ UIkit is a lightweight and modular front-end framework for developing fast and p
 You have the following options to get UIkit:
 
 - Download the [latest release](https://github.com/uikit/uikit/releases/latest) with pre-built CSS and JS.
-- Install with [Bower](https://bower.io) to get the pre-built CSS and JS, plus Less files to compile yourself. This is recommended when using UIkit for a typical web project: ```bower install uikit```
-- Clone the repo to get all source files including build scripts: `git clone git://github.com/uikit/uikit.git`
-- Install with [Npm](https://npmjs.com) to get all source files as they are available on Github: ```npm install uikit```
-- Directly load UIkit from [CDNJS](https://cdnjs.com): https://cdnjs.com/libraries/uikit
+- Install with [npm](https://npmjs.com) to get all source files as they are available on GitHub: ```npm install uikit```
+- Install with [yarn](https://yarnpkg.com/) to get all source files as they are available on GitHub: ```yarn add uikit```
+- Install with [pnpm](https://pnpm.io/) to get all source files as they are available on GitHub: ```pnpm add uikit```
+- Directly load UIkit from [jsDelivr](https://www.jsdelivr.com): https://www.jsdelivr.com/package/npm/uikit
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)